// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parseBody Error: 1`] = `
{
  "rawMessage": "Error:",
  "rawTrace": [],
}
`;

exports[`parseBody loader:1101:14) 1`] = `
{
  "rawMessage": "Error:",
  "rawTrace": [
    "    at G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:20:8",
    "    at new Promise (<anonymous>)",
    "    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:19:1)",
    "    at Module._compile (node:internal/modules/cjs/loader:1101:14)",
  ],
}
`;

exports[`parseErrorStack AggregateErrorExtra: ggregateError:
    at Object
    at Object.<anonymous> (G:\\Users\\The Project\\nodejs-yarn\\ws-error\\packages\\lazy-aggregate-error\\test\\temp.ts:9:9)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)
    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3) 1`] = `
ErrorStack {
  "code": undefined,
  "message": "ggregateError:
    at Object",
  "rawMessage": "AggregateErrorExtra: ggregateError:
    at Object",
  "rawStack": "AggregateErrorExtra: ggregateError:
    at Object
    at Object.<anonymous> (G:\\Users\\The Project\\nodejs-yarn\\ws-error\\packages\\lazy-aggregate-error\\test\\temp.ts:9:9)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)
    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3)",
  "rawTrace": [
    "    at Object.<anonymous> (G:\\Users\\The Project\\nodejs-yarn\\ws-error\\packages\\lazy-aggregate-error\\test\\temp.ts:9:9)",
    "    at Module._compile (node:internal/modules/cjs/loader:1101:14)",
    "    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)",
    "    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)",
    "    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)",
    "    at Module.load (node:internal/modules/cjs/loader:981:32)",
    "    at Function.Module._load (node:internal/modules/cjs/loader:822:12)",
    "    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)",
    "    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)",
    "    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3)",
  ],
  "traces": [
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "9",
      "indent": "    ",
      "line": "9",
      "source": "G:\\Users\\The Project\\nodejs-yarn\\ws-error\\packages\\lazy-aggregate-error\\test\\temp.ts",
    },
    {
      "callee": "Module._compile",
      "calleeNote": undefined,
      "col": "14",
      "indent": "    ",
      "line": "1101",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Module.m._compile",
      "calleeNote": undefined,
      "col": "23",
      "indent": "    ",
      "line": "1225",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts",
    },
    {
      "callee": "Module._extensions..js",
      "calleeNote": undefined,
      "col": "10",
      "indent": "    ",
      "line": "1153",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Object.require.extensions.<computed>",
      "calleeNote": "as .ts",
      "col": "12",
      "indent": "    ",
      "line": "1228",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts",
    },
    {
      "callee": "Module.load",
      "calleeNote": undefined,
      "col": "32",
      "indent": "    ",
      "line": "981",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Function.Module._load",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "822",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Function.executeUserEntryPoint",
      "calleeNote": "as runMain",
      "col": "12",
      "indent": "    ",
      "line": "79",
      "source": "node:internal/modules/run_main",
    },
    {
      "callee": "main",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "330",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts",
    },
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "3",
      "indent": "    ",
      "line": "477",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts",
    },
  ],
  "type": "AggregateErrorExtra",
}
`;

exports[`parseErrorStack Error: 
    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:31:16)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)
    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3) 1`] = `
ErrorStack {
  "code": undefined,
  "message": "",
  "rawMessage": "Error: ",
  "rawStack": "Error: 
    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:31:16)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)
    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3)",
  "rawTrace": [
    "    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:31:16)",
    "    at Module._compile (node:internal/modules/cjs/loader:1101:14)",
    "    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)",
    "    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)",
    "    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)",
    "    at Module.load (node:internal/modules/cjs/loader:981:32)",
    "    at Function.Module._load (node:internal/modules/cjs/loader:822:12)",
    "    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)",
    "    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)",
    "    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3)",
  ],
  "traces": [
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "16",
      "indent": "    ",
      "line": "31",
      "source": "G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts",
    },
    {
      "callee": "Module._compile",
      "calleeNote": undefined,
      "col": "14",
      "indent": "    ",
      "line": "1101",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Module.m._compile",
      "calleeNote": undefined,
      "col": "23",
      "indent": "    ",
      "line": "1225",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts",
    },
    {
      "callee": "Module._extensions..js",
      "calleeNote": undefined,
      "col": "10",
      "indent": "    ",
      "line": "1153",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Object.require.extensions.<computed>",
      "calleeNote": "as .ts",
      "col": "12",
      "indent": "    ",
      "line": "1228",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts",
    },
    {
      "callee": "Module.load",
      "calleeNote": undefined,
      "col": "32",
      "indent": "    ",
      "line": "981",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Function.Module._load",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "822",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Function.executeUserEntryPoint",
      "calleeNote": "as runMain",
      "col": "12",
      "indent": "    ",
      "line": "79",
      "source": "node:internal/modules/run_main",
    },
    {
      "callee": "main",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "330",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts",
    },
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "3",
      "indent": "    ",
      "line": "477",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts",
    },
  ],
  "type": "Error",
}
`;

exports[`parseErrorStack Error: foo:eval
    at eval (eval at <anonymous> (:1:1), <anonymous>:1:1)
    at <anonymous>:1:1 1`] = `
ErrorStack {
  "code": undefined,
  "message": "foo:eval",
  "rawMessage": "Error: foo:eval",
  "rawStack": "Error: foo:eval
    at eval (eval at <anonymous> (:1:1), <anonymous>:1:1)
    at <anonymous>:1:1",
  "rawTrace": [
    "    at eval (eval at <anonymous> (:1:1), <anonymous>:1:1)",
    "    at <anonymous>:1:1",
  ],
  "traces": [
    {
      "callee": "eval",
      "calleeNote": undefined,
      "col": "1",
      "eval": true,
      "evalCallee": "<anonymous>",
      "evalCalleeNote": undefined,
      "evalTrace": {
        "col": "1",
        "line": "1",
        "source": "<anonymous>",
      },
      "indent": "    ",
      "line": "1",
      "source": "",
    },
    {
      "callee": undefined,
      "col": "1",
      "indent": "    ",
      "line": "1",
      "source": "<anonymous>",
    },
  ],
  "type": "Error",
}
`;

exports[`parseErrorStack Error: foo:eval
    at eval (eval at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts:41:5), <anonymous>:1:1)
    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts:41:5)
    at Runtime._execModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:1394:24)
    at Runtime._loadModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:996:12)
    at Runtime.requireModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:828:12)
    at jestAdapter (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at runTestInternal (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\runTest.js:389:16)
    at runTest (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\runTest.js:481:34)
    at Object.worker (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\testWorker.js:133:12) 1`] = `
ErrorStack {
  "code": undefined,
  "message": "foo:eval",
  "rawMessage": "Error: foo:eval",
  "rawStack": "Error: foo:eval
    at eval (eval at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts:41:5), <anonymous>:1:1)
    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts:41:5)
    at Runtime._execModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:1394:24)
    at Runtime._loadModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:996:12)
    at Runtime.requireModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:828:12)
    at jestAdapter (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at runTestInternal (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\runTest.js:389:16)
    at runTest (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\runTest.js:481:34)
    at Object.worker (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\testWorker.js:133:12)",
  "rawTrace": [
    "    at eval (eval at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts:41:5), <anonymous>:1:1)",
    "    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts:41:5)",
    "    at Runtime._execModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:1394:24)",
    "    at Runtime._loadModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:996:12)",
    "    at Runtime.requireModule (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js:828:12)",
    "    at jestAdapter (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js:79:13)",
    "    at processTicksAndRejections (node:internal/process/task_queues:96:5)",
    "    at runTestInternal (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\runTest.js:389:16)",
    "    at runTest (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\runTest.js:481:34)",
    "    at Object.worker (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\testWorker.js:133:12)",
  ],
  "traces": [
    {
      "callee": "eval",
      "calleeNote": undefined,
      "col": "5",
      "eval": true,
      "evalCallee": "Object.<anonymous>",
      "evalCalleeNote": undefined,
      "evalTrace": {
        "col": "1",
        "line": "1",
        "source": "<anonymous>",
      },
      "indent": "    ",
      "line": "41",
      "source": "G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts",
    },
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "5",
      "indent": "    ",
      "line": "41",
      "source": "G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts",
    },
    {
      "callee": "Runtime._execModule",
      "calleeNote": undefined,
      "col": "24",
      "indent": "    ",
      "line": "1394",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js",
    },
    {
      "callee": "Runtime._loadModule",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "996",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js",
    },
    {
      "callee": "Runtime.requireModule",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "828",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runtime\\build\\index.js",
    },
    {
      "callee": "jestAdapter",
      "calleeNote": undefined,
      "col": "13",
      "indent": "    ",
      "line": "79",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-circus\\build\\legacy-code-todo-rewrite\\jestAdapter.js",
    },
    {
      "callee": "processTicksAndRejections",
      "calleeNote": undefined,
      "col": "5",
      "indent": "    ",
      "line": "96",
      "source": "node:internal/process/task_queues",
    },
    {
      "callee": "runTestInternal",
      "calleeNote": undefined,
      "col": "16",
      "indent": "    ",
      "line": "389",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\runTest.js",
    },
    {
      "callee": "runTest",
      "calleeNote": undefined,
      "col": "34",
      "indent": "    ",
      "line": "481",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\runTest.js",
    },
    {
      "callee": "Object.worker",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "133",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jest\\node_modules\\jest-runner\\build\\testWorker.js",
    },
  ],
  "type": "Error",
}
`;

exports[`parseEvalSource error-stack.test.ts:41:5), <anonymous>:1:1 1`] = `
{
  "col": "5",
  "evalCallee": "Object.<anonymous>",
  "evalCalleeNote": undefined,
  "evalTrace": {
    "col": "1",
    "line": "1",
    "source": "<anonymous>",
  },
  "indent": "",
  "line": "41",
  "source": "G:\\Users\\The Project\\fork\\error-stack\\test\\error-stack.test.ts",
}
`;

exports[`parseMessage Error:  1`] = `
{
  "code": undefined,
  "message": "",
  "type": "Error",
}
`;

exports[`parseMessage QmSJF5DHa4
XogNVQh1AH1UTGM9QN1ZqAywEgvPrfPP9f28'" 1`] = `
{
  "code": "ERR_ASSERTION",
  "message": "'CID(QmQndiGjG2YFoUfYPtLsy3RcTxPVuFRmEZFE2WAEtSYh3x)' isSameCID "'/ipfs/QmSJF5DHa4
XogNVQh1AH1UTGM9QN1ZqAywEgvPrfPP9f28'"",
  "type": "AssertionError",
}
`;

exports[`parseSource bin.ts:330:12 1`] = `
{
  "col": "12",
  "line": "330",
  "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts",
}
`;

exports[`parseSource bin.ts:477:3 1`] = `
{
  "col": "3",
  "line": "477",
  "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts",
}
`;

exports[`parseSource index.ts:1225:23 1`] = `
{
  "col": "23",
  "line": "1225",
  "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts",
}
`;

exports[`parseSource index.ts:1228:12 1`] = `
{
  "col": "12",
  "line": "1228",
  "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts",
}
`;

exports[`parseSource loader:822:12 1`] = `
{
  "col": "12",
  "line": "822",
  "source": "node:internal/modules/cjs/loader",
}
`;

exports[`parseSource loader:981:32 1`] = `
{
  "col": "32",
  "line": "981",
  "source": "node:internal/modules/cjs/loader",
}
`;

exports[`parseSource loader:1101:14 1`] = `
{
  "col": "14",
  "line": "1101",
  "source": "node:internal/modules/cjs/loader",
}
`;

exports[`parseSource loader:1153:10 1`] = `
{
  "col": "10",
  "line": "1153",
  "source": "node:internal/modules/cjs/loader",
}
`;

exports[`parseSource run_main:79:12 1`] = `
{
  "col": "12",
  "line": "79",
  "source": "node:internal/modules/run_main",
}
`;

exports[`parseSource temp.ts:6:13 1`] = `
{
  "col": "13",
  "line": "6",
  "source": "G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts",
}
`;

exports[`parseStack bin.ts:477:3) 1`] = `
{
  "code": undefined,
  "message": "ggregateError:
    at Object.<anonymous> (<anonymous>:19:1)",
  "rawMessage": "AggregateError: ggregateError:
    at Object.<anonymous> (<anonymous>:19:1)",
  "rawStack": "AggregateError: ggregateError:
    at Object.<anonymous> (<anonymous>:19:1)
    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:11:9)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)
    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3)",
  "rawTrace": [
    "    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:11:9)",
    "    at Module._compile (node:internal/modules/cjs/loader:1101:14)",
    "    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)",
    "    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)",
    "    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)",
    "    at Module.load (node:internal/modules/cjs/loader:981:32)",
    "    at Function.Module._load (node:internal/modules/cjs/loader:822:12)",
    "    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)",
    "    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)",
    "    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3)",
  ],
  "traces": [
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "9",
      "indent": "    ",
      "line": "11",
      "source": "G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts",
    },
    {
      "callee": "Module._compile",
      "calleeNote": undefined,
      "col": "14",
      "indent": "    ",
      "line": "1101",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Module.m._compile",
      "calleeNote": undefined,
      "col": "23",
      "indent": "    ",
      "line": "1225",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts",
    },
    {
      "callee": "Module._extensions..js",
      "calleeNote": undefined,
      "col": "10",
      "indent": "    ",
      "line": "1153",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Object.require.extensions.<computed>",
      "calleeNote": "as .ts",
      "col": "12",
      "indent": "    ",
      "line": "1228",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts",
    },
    {
      "callee": "Module.load",
      "calleeNote": undefined,
      "col": "32",
      "indent": "    ",
      "line": "981",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Function.Module._load",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "822",
      "source": "node:internal/modules/cjs/loader",
    },
    {
      "callee": "Function.executeUserEntryPoint",
      "calleeNote": "as runMain",
      "col": "12",
      "indent": "    ",
      "line": "79",
      "source": "node:internal/modules/run_main",
    },
    {
      "callee": "main",
      "calleeNote": undefined,
      "col": "12",
      "indent": "    ",
      "line": "330",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts",
    },
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "3",
      "indent": "    ",
      "line": "477",
      "source": "C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts",
    },
  ],
  "type": "AggregateError",
}
`;

exports[`parseStack example.js:3:13) 1`] = `
{
  "code": undefined,
  "message": "
    Error: foo",
  "rawMessage": "AggregateError: 
    Error: foo",
  "rawStack": "AggregateError: 
    Error: foo
        at Object.<anonymous> (/Users/sindresorhus/dev/aggregate-error/example.js:3:33)
    Error: bar
        at Object.<anonymous> (/Users/sindresorhus/dev/aggregate-error/example.js:3:13)
    Error: baz
        at Object.<anonymous> (/Users/sindresorhus/dev/aggregate-error/example.js:3:13)
    at AggregateError (/Users/sindresorhus/dev/aggregate-error/index.js:19:3)
    at Object.<anonymous> (/Users/sindresorhus/dev/aggregate-error/example.js:3:13)",
  "rawTrace": [
    "        at Object.<anonymous> (/Users/sindresorhus/dev/aggregate-error/example.js:3:33)",
    "    Error: bar",
    "        at Object.<anonymous> (/Users/sindresorhus/dev/aggregate-error/example.js:3:13)",
    "    Error: baz",
    "        at Object.<anonymous> (/Users/sindresorhus/dev/aggregate-error/example.js:3:13)",
    "    at AggregateError (/Users/sindresorhus/dev/aggregate-error/index.js:19:3)",
    "    at Object.<anonymous> (/Users/sindresorhus/dev/aggregate-error/example.js:3:13)",
  ],
  "traces": [
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "33",
      "indent": "        ",
      "line": "3",
      "source": "/Users/sindresorhus/dev/aggregate-error/example.js",
    },
    {
      "indent": "    ",
      "raw": true,
      "rawLine": "Error: bar",
    },
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "13",
      "indent": "        ",
      "line": "3",
      "source": "/Users/sindresorhus/dev/aggregate-error/example.js",
    },
    {
      "indent": "    ",
      "raw": true,
      "rawLine": "Error: baz",
    },
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "13",
      "indent": "        ",
      "line": "3",
      "source": "/Users/sindresorhus/dev/aggregate-error/example.js",
    },
    {
      "callee": "AggregateError",
      "calleeNote": undefined,
      "col": "3",
      "indent": "    ",
      "line": "19",
      "source": "/Users/sindresorhus/dev/aggregate-error/index.js",
    },
    {
      "callee": "Object.<anonymous>",
      "calleeNote": undefined,
      "col": "13",
      "indent": "    ",
      "line": "3",
      "source": "/Users/sindresorhus/dev/aggregate-error/example.js",
    },
  ],
  "type": "AggregateError",
}
`;

exports[`parseStack temp.ts:42:2 1`] = `
{
  "code": "ERR_ASSERTION",
  "message": "'CID(QmQndiGjG2YFoUfYPtLsy3RcTxPVuFRmEZFE2WAEtSYh3x)' isSameCID "'/ipfs/QmSJF5DHa4
XogNVQh1AH1UTGM9QN1ZqAywEgvPrfPP9f28'"",
  "rawMessage": "AssertionError [ERR_ASSERTION]: 'CID(QmQndiGjG2YFoUfYPtLsy3RcTxPVuFRmEZFE2WAEtSYh3x)' isSameCID "'/ipfs/QmSJF5DHa4
XogNVQh1AH1UTGM9QN1ZqAywEgvPrfPP9f28'"",
  "rawStack": "AssertionError [ERR_ASSERTION]: 'CID(QmQndiGjG2YFoUfYPtLsy3RcTxPVuFRmEZFE2WAEtSYh3x)' isSameCID "'/ipfs/QmSJF5DHa4
XogNVQh1AH1UTGM9QN1ZqAywEgvPrfPP9f28'"
    at G:\\Users\\The Project\\p2p\\ws-ipfs\\packages\\@lazy-ipfs\\compatible-files\\lib\\cp.ts:93:51
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at G:\\Users\\The Project\\p2p\\ws-ipfs\\packages\\@lazy-ipfs\\compatible-files\\test\\temp.ts:42:2",
  "rawTrace": [
    "    at G:\\Users\\The Project\\p2p\\ws-ipfs\\packages\\@lazy-ipfs\\compatible-files\\lib\\cp.ts:93:51",
    "    at processTicksAndRejections (node:internal/process/task_queues:96:5)",
    "    at G:\\Users\\The Project\\p2p\\ws-ipfs\\packages\\@lazy-ipfs\\compatible-files\\test\\temp.ts:42:2",
  ],
  "traces": [
    {
      "callee": undefined,
      "col": "51",
      "indent": "    ",
      "line": "93",
      "source": "G:\\Users\\The Project\\p2p\\ws-ipfs\\packages\\@lazy-ipfs\\compatible-files\\lib\\cp.ts",
    },
    {
      "callee": "processTicksAndRejections",
      "calleeNote": undefined,
      "col": "5",
      "indent": "    ",
      "line": "96",
      "source": "node:internal/process/task_queues",
    },
    {
      "callee": undefined,
      "col": "2",
      "indent": "    ",
      "line": "42",
      "source": "G:\\Users\\The Project\\p2p\\ws-ipfs\\packages\\@lazy-ipfs\\compatible-files\\test\\temp.ts",
    },
  ],
  "type": "AssertionError",
}
`;

exports[`parseTrace     at new Promise (<anonymous>) 1`] = `
{
  "callee": "new Promise",
  "calleeNote": undefined,
  "indent": "    ",
  "source": "<anonymous>",
}
`;

exports[`parseTrace     at new Promise (<anonymous>) 2`] = `"    at new Promise (<anonymous>)"`;

exports[`parseTrace core.js:100:15) 1`] = `
{
  "callee": "Object.errorHandler",
  "calleeNote": "as handleError",
  "col": "15",
  "indent": "        ",
  "line": "100",
  "source": "G:/Users/The Project/p2p/ws-ipfs/node_modules/ipfs-http-client/src/lib/core.js",
}
`;

exports[`parseTrace core.js:100:15) 2`] = `"        at Object.errorHandler [as handleError] (G:/Users/The Project/p2p/ws-ipfs/node_modules/ipfs-http-client/src/lib/core.js:100:15)"`;

exports[`stringifyErrorStack {
  type: 'AggregateErrorExtra',
  message: 'ggregateError: \\nat Object',
  rawTrace: [
    '    at Object.<anonymous> (G:\\\\Users\\\\The Project\\\\fork\\\\error-stack\\\\test\\\\temp.ts:11:9)',
    '    at Module._compile (node:internal/modules/cjs/loader:1101:14)',
    '    at Module.m._compile (C:\\\\Users\\\\User\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\ts-node\\\\src\\\\index.ts:1225:23)',
    '    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)',
    '    at Object.require.extensions.<computed> [as .ts] (C:\\\\Users\\\\User\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\ts-node\\\\src\\\\index.ts:1228:12)',
    '    at Module.load (node:internal/modules/cjs/loader:981:32)',
    '    at Function.Module._load (node:internal/modules/cjs/loader:822:12)',
    '    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)',
    '    at main (C:\\\\Users\\\\User\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\ts-node\\\\src\\\\bin.ts:330:12)',
    '    at Object.<anonymous> (C:\\\\Users\\\\User\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\ts-node\\\\src\\\\bin.ts:477:3)'
  ]
} 1`] = `
"AggregateErrorExtra: ggregateError: 
at Object
    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:11:9)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)
    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3)"
`;

exports[`stringifyErrorStack {
  type: 'AggregateErrorExtra',
  message: 'ggregateError: \\nat Object',
  traces: [
    {
      callee: 'Object.<anonymous>',
      calleeNote: undefined,
      source: 'G:\\\\Users\\\\The Project\\\\fork\\\\error-stack\\\\test\\\\temp.ts',
      line: '11',
      col: '9',
      indent: '    '
    },
    {
      callee: 'Module._compile',
      calleeNote: undefined,
      source: 'node:internal/modules/cjs/loader',
      line: '1101',
      col: '14',
      indent: '    '
    },
    {
      callee: 'Module.m._compile',
      calleeNote: undefined,
      source: 'C:\\\\Users\\\\User\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\ts-node\\\\src\\\\index.ts',
      line: '1225',
      col: '23',
      indent: '    '
    },
    {
      callee: 'Module._extensions..js',
      calleeNote: undefined,
      source: 'node:internal/modules/cjs/loader',
      line: '1153',
      col: '10',
      indent: '    '
    },
    {
      callee: 'Object.require.extensions.<computed>',
      calleeNote: 'as .ts',
      source: 'C:\\\\Users\\\\User\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\ts-node\\\\src\\\\index.ts',
      line: '1228',
      col: '12',
      indent: '    '
    },
    {
      callee: 'Module.load',
      calleeNote: undefined,
      source: 'node:internal/modules/cjs/loader',
      line: '981',
      col: '32',
      indent: '    '
    },
    {
      callee: 'Function.Module._load',
      calleeNote: undefined,
      source: 'node:internal/modules/cjs/loader',
      line: '822',
      col: '12',
      indent: '    '
    },
    {
      callee: 'Function.executeUserEntryPoint',
      calleeNote: 'as runMain',
      source: 'node:internal/modules/run_main',
      line: '79',
      col: '12',
      indent: '    '
    },
    {
      callee: 'main',
      calleeNote: undefined,
      source: 'C:\\\\Users\\\\User\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\ts-node\\\\src\\\\bin.ts',
      line: '330',
      col: '12',
      indent: '    '
    },
    {
      callee: 'Object.<anonymous>',
      calleeNote: undefined,
      source: 'C:\\\\Users\\\\User\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\ts-node\\\\src\\\\bin.ts',
      line: '477',
      col: '3',
      indent: '    '
    }
  ]
} 1`] = `
"AggregateErrorExtra: ggregateError: 
at Object
    at Object.<anonymous> (G:\\Users\\The Project\\fork\\error-stack\\test\\temp.ts:11:9)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module.m._compile (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1225:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\index.ts:1228:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at main (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:330:12)
    at Object.<anonymous> (C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\ts-node\\src\\bin.ts:477:3)"
`;
